tests/test_tebd.py

```diff
@@ -109,3 +127,5 @@ def test_RandomUnitaryEvolution():
     eng.run()
     print(eng.psi.chi)
     assert tuple(eng.psi.chi) == (16, 8)
+
+
```

Comment:
add a test for _eig_based_svd() or at least a big, fat warning in the functions doc string  (and better an `raise NotImplementedError` in the function code if we don't fix it for now).

Reply:
You only encounter this if you set the non-default option `use_eig_based_svd=True` in the `QRBasedTebdEngine`s config. Have documented a small non-fat warning there. Or do you want a fatter warning?

Reply:
looks good to me now.